### ISSUE TITLE
Not sending commands to stata correctly

### ISSUE THREAD

**Issue by kb12344** [Hemken/Statamarkdown#5]
Using RStudio Version Version 1.1.463 and Stata16 and the following code:
\```{stata, engine.path="C:/Program Files/Stata16/StataIC-64", comment=""}
sysuse auto
summarize
\```

I receive this error in stata:
. C:\Users\kb289\AppData\Local\Temp\RtmpCGQRnq\chunk-code-227c3e2b33.txt 
command C is unrecognized

Trying this is stata works, but I couldn't figure out how to get markdown to send the command like this:
do "C:\Users\kb289\AppData\Local\Temp\RtmpCGQRnq\chunk-code-227c3e2b33.txt"

**Comment by Hemken** [comment]
What versions of things are you running?  My test file is

++++++++++++++++++++++
---
title: "Stata test"
author: "Doug Hemken"
date: "8/12/2019"
output: html_document
---

```{r setup, include=FALSE}
library(Statamarkdown)
```

## Not sending commands to stata correctly

```{r version}
R.version.string
packageVersion("Statamarkdown")
packageVersion("knitr")
```

```{stata, engine.path="C:/Program Files/Stata16/StataSE-64", comment=""}
sysuse auto
summarize
```
++++++++++++++++++++++++

And I run this by opening the file in Rstudio and clicking the “Knit” button or keying Ctrl-Shift-K.

I get no error.

**Comment by Hemken** [comment]
My R Markdown log looks like this:

+++++++++++++++++++++++++++++


processing file: Stata16test.RMD
  |...........                                                      |  17%
  ordinary text without R code

  |......................                                           |  33%
label: setup (with options)
List of 1
$ include: logi FALSE

  |................................                                 |  50%
  ordinary text without R code

  |...........................................                      |  67%
label: version
  |......................................................           |  83%
  ordinary text without R code

  |.................................................................| 100%
label: unnamed-chunk-1 (with options)
List of 3
$ engine.path: chr "C:/Program Files/Stata16/StataSE-64"
$ comment    : chr ""
$ engine     : chr "stata"

R output from version
R output from version
R output from version
running: C:/Program Files/Stata16/StataSE-64  /q /e do "U:\stata2da82d6f26df.do"
character(0)
stata output from unnamed-chunk-1

output file: Stata16test.knit.md

"C:/PROGRA~1/Pandoc/pandoc" +RTS -K512m -RTS Stata16test.utf8.md --to html4 --from markdown+autolink_bare_uris+ascii_identifiers+tex_math_single_backslash+smart --output Stata16test.html --email-obfuscation none --self-contained --standalone --section-divs --template "C:\PROGRA~1\R\R-36~1.1\SITE-L~1\RMARKD~1\rmd\h\DEFAUL~1.HTM" --no-highlight --variable highlightjs=1 --variable "theme:bootstrap" --include-in-header "C:\Users\hemken\AppData\Local\Temp\Rtmpe8vX1k\rmarkdown-str2da8776b2bdc.html" --mathjax --variable "mathjax-url:https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"

Output created: Stata16test.html
++++++++++++++++++++++++++++++++++++++++

**Comment by kb12344** [comment]
Oh I just realized that it works when I click knit to create the whole document.  It just wasn't working when trying to run the chunk individually.   

R version 3.6.0 (2019-04-26)
> packageVersion("Statamarkdown")
[1] ‘0.3.8’
> packageVersion("knitr")
[1] ‘1.23’

**Comment by Hemken** [comment]
I see.  I don’t think any languages other than R will work in notebook mode, at least not currently.
